pkg/controller/nodelifecycle/scheduler/taint_manager.go

```diff
@@ -185,8 +185,9 @@ func NewNoExecuteTaintManager(c clientset.Interface, getPod GetPodFunc, getNode
 		getNode:      getNode,
 		taintedNodes: make(map[string][]v1.Taint),
 
-		nodeUpdateQueue: workqueue.New(),
-		podUpdateQueue:  workqueue.New(),
+		// TODO(mtaufen): okay these are getting really verbose...
+		nodeUpdateQueue: workqueue.NewNamed("node_lifecycle_controller_noexec_taint_node"),
```

Comment:
Isn't `noexec_taint_node` and `noexec_taint_pod` descriptive enough? I suspect that those two suffixes are actually already uniquely identifying.